docs/reference/extensions/ros-2-content-extensions.rst

```diff
@@ -0,0 +1,73 @@
+.. _ros-2-content-extensions:
+
+ROS 2 Content extensions
+========================
+
+The ROS 2 Content extensions comprise the main ROS 2 extensions, plus additional
+settings to enable `content sharing <https://snapcraft.io/docs/content-interface>`_.
+
+These extensions are split across ROS 2 versions and content types, and are declared
+with the format ``ros2-<version>-<metapackage>``. The available extensions are:
+
+- ``ros2-foxy-ros-core``
+- ``ros2-foxy-ros-base``
+- ``ros2-foxy-desktop``
+- ``ros2-humble-ros-core``
+- ``ros2-humble-ros-base``
+- ``ros2-humble-desktop``
+- ``ros2-jazzy-ros-core``
+- ``ros2-jazzy-ros-base``
+- ``ros2-jazzy-desktop``
```

Comment:
I tested for Jazzy, and all three had identical outputs. Am I missing something?

Reply:
They shouldn't be identical. Based on the source they should:

1. Refer to a different content plug (`ros-jazzy-ros-(core|base|desktop)`)
2. Include a different build snap in any part using the `colcon` plugin
3. Name the added part differently.

But yeah, the structure should be identical otherwise.

Reply:
OK, not quite identical, but aside from some string composition, they do nothing drastically different. Which makes the proposition of including samples for all three quite low.